target/linux/ar71xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -138,6 +138,8 @@ case "$FIRMWARE" in
 	archer-c59-v1|\
 	archer-c60-v1)
 		ath10kcal_extract "art" 20480 12064
+		ln -sf /lib/firmware/ath10k/pre-cal-pci-0000\:00\:00.0.bin \
```

Comment:
@chunkeey, @heinzek, +1 from me for fixing "ath10k-firmware: add qca9888 firmware" and extract board.bin from flash instead of ln hack (even if it actually works). Or, I would wait for fix in upstream (ath10k-firmware).